net/unbound/Makefile

```diff
@@ -43,7 +43,8 @@ define Package/unbound
 endef
 
 define Package/unbound/description
-  This package contains the Unbound daemon.
+  This package contains the Unbound daemon. 
+  Please see https://github.com/openwrt/packages/blob/master/net/unbound/files/README.md
```

Comment:
Other packages do this reference to README.md for UCI and other OpenWRT specific content (not found upstream). This will appear in the Luci software install tool and people can find UCI requirements easier.